src/platform/PICO/dma_pico.c

```diff
@@ -47,7 +47,7 @@ dmaChannelDescriptor_t dmaDescriptors[DMA_LAST_HANDLER] = {
     DEFINE_DMA_CHANNEL(DMA_CH9_HANDLER),
     DEFINE_DMA_CHANNEL(DMA_CH10_HANDLER),
     DEFINE_DMA_CHANNEL(DMA_CH11_HANDLER),
-#ifdef RP2350
+#if defined(RP2350A) || defined(RP2350B)
```

Comment:
This can be reverted now

Reply:
Yes - this was because RP2350 was not defined as the family. It will be reverted to RP2350

Reply:
RP2040... or an RP2450 with more than 16 channels... ;)